ydb/src/credentials.rs

```diff
@@ -228,3 +235,48 @@ impl Credentials for GCEMetadata {
         format!("GoogleComputeEngineMetadata from {}", self.uri.as_str())
     }
 }
+
+pub struct UserPasswordAuth {
```

Comment:
Better to rename the structure to StaticCredentialsAuth - by auth type name (we named it static credentials)